examples/graph/requires/consume_cmake_macro.rst

```diff
@@ -0,0 +1,77 @@
+Use a CMake macro packaged in a dependency
+------------------------------------------
+
+When a package recipe want to provide a CMake functionality via a macro, it can be done as follows.
```

Comment:
```suggestion
When a package recipe want to provide a CMake functionality via a macro, it can be done as follows.
```
```suggestion
When a package recipe wants to provide a CMake functionality via a macro, it can be done as follows.
```